onnx2keras/padding_layers.py

```diff
@@ -13,36 +14,53 @@ def convert_padding(node, params, layers, node_name, keras_name):
     :return: None
     """
     # It's binary by-default
+    logger = logging.getLogger("onnx2keras:padding")
     params['mode'] = params['mode'].decode('ascii')
     input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
 
+    pads = params['pads']
+
     if params['mode'] == 'constant':
         # raise AssertionError('Cannot convert non-constant padding')
 
         if params['value'] != 0.0:
             raise AssertionError('Cannot convert non-zero padding')
 
         # Magic ordering
-        padding_layer = keras.layers.ZeroPadding2D(
-            padding=((params['pads'][2], params['pads'][6]), (params['pads'][3], params['pads'][7])),
-            name=keras_name
-        )
-
+        if len(pads) == 8:
+            padding_layer = keras.layers.ZeroPadding2D(
+                padding=((pads[2], pads[6]), (pads[3], pads[7])),
+                name=keras_name
+            )
+        else:
+            logger.warning("Caution - no test yet")
+            padding_layer = keras.layers.ZeroPadding3D(
+                padding=((pads[2], pads[7]), (pads[3], pads[8]), (pads[4], pads[9])),
+                name=keras_name
+            )
         layers[node_name] = padding_layer(input_0)
     elif params['mode'] == 'reflect':
 
-        def target_layer(x, pads=params['pads']):
+        def target_layer(x):
             import tensorflow as tf
-            layer = tf.pad(x, [[0, 0], [0, 0], [pads[2], pads[6]], [pads[3], pads[7]]], 'REFLECT')
+            if len(pads)==8:
```

Comment:
Please, use spaces between comparison operation. I'm about to fix all lint issues I have in this project